core/base/inc/TColor.h

```diff
@@ -62,8 +62,8 @@ class TColor : public TNamed {
    Float_t       GetSaturation() const { return IsGrayscale() ? 0 : fSaturation; }
    Float_t       GetAlpha() const { return fAlpha; }
    virtual Float_t GetGrayscale() const { /*ITU*/ return 0.299f*fRed + 0.587f*fGreen + 0.114f*fBlue; }
-   virtual void  ls(Option_t *option="") const;
-   virtual void  Print(Option_t *option="") const;
+   void  ls(Option_t *option="") const override;
+   void  Print(Option_t *option="") const override;
    virtual void  SetAlpha(Float_t a) { fAlpha = a; }
```

Comment:
```suggestion
   void          ls(Option_t *option="") const override;
   void          Print(Option_t *option="") const override;
   virtual void  SetAlpha(Float_t a) { fAlpha = a; }
```

This (adding override) is a great improvement.   A small request: the intended code layout is to align the member function names.

Reply:
I squashed the suggested change into my commit.